ui/app/src/components/PathNavigatorTree/PathNavigatorTree.tsx

```diff
@@ -181,12 +199,19 @@ export default function PathNavigatorTree(props: PathNavigatorTreeProps) {
     }
   }, [site, user.username, id, dispatch, rootPath, excludes, limit, state, uiConfig.currentSite, storedState]);
 
+  useMount(() => {
+    if (state) {
+      state.expanded.forEach((path) => fetchingPathsRef.current.push(path));
+      dispatch(pathNavigatorTreeBackgroundRefresh({ id }));
+    }
+  });
+
```

Comment:
can we have as part of same effect that initialises?

Reply:
no, because the other effect is listening the state, and it changes very often